codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/SymbolVisitor.java

```diff
@@ -222,7 +223,11 @@ private String getDefaultShapeName(Shape shape) {
     }
 
     private String getDefaultMemberName(MemberShape shape) {
-        return StringUtils.capitalize(removeLeadingUnderscores(shape.getMemberName()));
+        String memberName =  StringUtils.capitalize(removeLeadingUnderscores(shape.getMemberName()));
+        if (shape.hasTrait(UnexportedMemberTrait.class)) {
+            memberName = Character.toLowerCase(memberName.charAt(0)) + memberName.substring(1);
+        }
```

Comment:
Just to verify, do we have validations for member shape name to not have numeric's in name? 

example case : `__123Alpha`

Reply:
Most likely not, protocol tests should be created for these, though the test probably have to be specific to the SDK.

It looks like this needs a little rework to make sure the lower casing happens after numbers have been stripped. off.  I'm guessing Smtihy allows member/shapes named `__123abc` :(

Reply:
yeah according to smithy's docs `__123abc` is a valid shape id :( https://awslabs.github.io/smithy/1.0/spec/core/model.html?highlight=membername#shape-id

Reply:
Ended up refactoring several places that where symbols weren't being used, and added case to the smithy-go tests to ensure the sdk handles `__123abc` correctly.